syft/frameworks/torch/nn/__init__.py

```diff
@@ -1,5 +1,5 @@
 from syft.frameworks.torch.nn.conv import Conv2d
-from syft.frameworks.torch.nn.functional import conv2d
+from syft.frameworks.torch.nn.functional import conv2d, maxpool2d, avgpool2d
```

Comment:
```suggestion
from syft.frameworks.torch.nn.functional import conv2d
from syft.frameworks.torch.nn.functional import maxpool2d
from syft.frameworks.torch.nn.functional import avgpool2d
```
Just to be consistent with `__init__.py` files eveywhere

Reply:
why do you think that's necessary? is there a use case that benefits from that?

Reply:
Doesn't change the functionality in anyway. It's just conventional to do so 😄